cluster/Vagrantfile

```diff
@@ -75,36 +92,65 @@ $1
 EOF
 SCRIPT
 
-# begin execution here
+# Begin execution here
 num_nodes = ENV['CONTIV_NODES'] ? ENV['CONTIV_NODES'].to_i : 2
+num_workers = num_nodes - 1
 base_ip = ENV['CONTIV_IP_PREFIX'] || '192.168.2.'
-start = ENV['CONTIV_KUBEADM'] ? 50 : 50 + num_nodes
-name_start = ENV['CONTIV_KUBEADM'] ? 1 : 1 + num_nodes
-node_ips = Array.new(num_nodes) { |n| base_ip + (n + start).to_s }
-node_names = Array.new(num_nodes) { |n| "contiv-node#{n + name_start}" }
+num_orcs = orchestrators.length
+ip = 50
 
-master_ip = create_etc_hosts(num_nodes, base_ip, start)
-create_cfg_info(num_nodes, node_ips)
+node_names = []
+node_ips = []
+node_orcs = []
+master_ips = []
+
+orchestrators.each do |o|
+  # Save the start location in IP array
+  start = node_ips.length
+
+  node_names.push(o + '-master')
+  num_workers.times do |n|
+    node_names.push(o + '-worker' + n.to_s)
+  end
+  master_ips.push(base_ip + ip.to_s)
+  num_nodes.times do |_n|
```

Comment:
You can completely omit the `|vars|` if you don't use any of them, e.g., `5.times { puts "hi" }`